docs/source/conf.py

```diff
@@ -89,6 +89,18 @@
 # The full version, including alpha/beta/rc tags.
 release = cartopy.__version__
 
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../lib/cartopy/examples'],
+    'filename_pattern': '^((?!sgskip).)*$',
```

Comment:
ah I see - I wonder if this is just because I'm working on a relatively underpowered laptop...I'll remove the sgskip and will add some print statements so it doesn't look like it's hanging